models/staging/stg_tripadvisor__review.sql

```diff
@@ -0,0 +1,36 @@
+{{ config(materialized='table') }}
+
+WITH tripadvisor__review__source_name AS (
+    SELECT *
+    FROM {{ source('raw_tripadvisor', 'source_tripadvisor__scrape_info') }}
+)
+
+, tripadvisor__review__rename_column AS (
+    SELECT 
+        location_id
+        , location_url
+        , address_from_url AS location_address
+        , google_maps_link AS location_map
+        , lat AS latitude
+        , long AS longitude
+        , tel AS phone_number
+        , open_hour AS open_hour
+        , price_range 
+        , c.element AS cuisine
+        , ranking AS location_rank
+        , rating AS location_overall_rate
+        , review_count 
+        , review_count_scraped
+        , e.element.rating AS location_rating
+        , e.element.review_date AS review_date
+        , e.element.review_type AS review_type
+        , e.element.text AS review_description
+        , e.element.title AS review_title
+        , e.element.user AS user
+    FROM tripadvisor__review__source_name
+    , UNNEST(reviews.list) AS e
+    , UNNEST(cuisine.list) AS c
```

Comment:
Hi @ngynzyy, when i run scraping for the web, i realized that some restaurant even have reviews but not the cuisine like this

![image](https://github.com/user-attachments/assets/c46a75db-c1eb-4a4b-b93d-cfd4b801086d)

Will this cause no rows in the fact tables even it has reviews

Reply:
Maybe you can check it for me, the location id is `12664198`, i've already uploaded the new data, you can rerun the `dbt run` to check it